### ISSUE TITLE
RimWorld.Planet.TileTemperaturesCompSystem

### ISSUE THREAD

**Issue by Firebirdss80** [cseelhoff/RimThreaded#421]
**Describe the bug**
I Have no clue

**To Reproduce**
1. Open Save
2. Click on play
3. See error
4. Profit!

**Error Log**
https://gist.github.com/e150effe8d2f187c9b156d328ecb14ba

**Comment by Firebirdss80** [comment]
[New Arrivals2.zip](https://github.com/cseelhoff/RimThreaded/files/5897082/New.Arrivals2.zip)

**Comment by bababooeyy** [comment]
Can confirm that this happens after 25 days for me on the given save, reducing TPS to 0 and making game unplayable. 

The Bug Logs:
https://gist.github.com/aa69a8ef9aa08f6d37be66fdc9941fa6

The Save with the bug active so you dont have to sit through 25 days of time:
https://drive.google.com/file/d/1_SPMj9eyqIVXYvSqByJ8Pd0gQ7POC368/view?usp=sharing


However. I was unable to reproduce these errors on a regular-sized world (30% coverage) with 20 by 20 map size using Better Map Sizes after 45 days. 


I was, however, able to get a null reference exception bug on day 21 with a 20 by 20 map size on a dev-size world (5% coverage). It does not seem to be related to the above mentioned error, and does not affect tps, but i just thought i would mention it as it looked similar.

https://gist.github.com/92884cb772a65347b254eea3bd7a66d5

After a further 20 days of testing on the 5% coverage save, i did not get any further errors, and decided to move on. However, this may not be adequate testing, and i will be uploading the save if anyone wants to continue testing it. 

https://drive.google.com/file/d/1cM1yk4yRNu96Gv-kxw30BFsCZPZ2a_QC/view?usp=sharing


There were two minor, routine 'tried to reserve job, but was already reserved by him' errors that I wont upload the logs to. 

There was also an error i had never seen before, that i was unable to reproduce. It happened when a rabbit went mad and attacked my colonist.  **Please tell me if i should make a new issue for this**

Error Log:
https://gist.github.com/9d1caa38035edc238bd1fa509a893ef6 

Save File:
https://drive.google.com/file/d/125IQET6Pmfx3p2gr8ejdZqhLVVh6YYBd/view?usp=sharing


In conclusion, I was able to reproduce the error you described on the save that you provided, but not on any saves of my own. I either am missing something in my testing(very likely lol), or there is something special about your save that I cannot as of yet pinpoint. 

Please ping me on the discord or inform me here if i have made any errors in my testing or if you have doubts on my process, i am very new to testing this mod and am open to suggestions on how to improve my testing.

**Comment by thefriendlycacti** [comment]
This has been around for a while https://github.com/cseelhoff/RimThreaded/issues/393

**Comment by cseelhoff** [comment]
fixed in 1.3.0.9
